eng/pipelines/templates/steps/vmr-pull-updates.yml

```diff
@@ -34,5 +34,7 @@ steps:
     --tmp $(Agent.TempDirectory)
     --branch ${{ parameters.vmrBranch }}
     --target-ref ${{ parameters.targetRef }}
+    --readme-template $(Agent.BuildDirectory)/installer/src/VirtualMonoRepo/README.template.md
```

Comment:
I could also hardcode them in the script file but I thought it's clearer like this, not sure if it's ok?

Reply:
I think this is better as the script is invoked in other scenarios as well